### PR TITLE
Added a question on recursion in general-cs-quiz.ts

### PR DESCRIPTION
**Pull request by deedeecee**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1224,6 +1224,18 @@ const generalCSQuiz = [
       "Big theta is either the exact performance value of the algorithm, or a useful range between narrow upper and lower bounds.",
     Link: "https://www.freecodecamp.org/news/big-theta-and-asymptotic-notation-explained/",
   },
+  
+  {
+    Question: "For a valid algorithm that uses recursion with 'b' branches and has 'd' depth in the recursion tree, the Big-O time complexity can often (but not always) be calculated using which formula?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1229]: It is best to keep the questions on the shorter side so it looks better when rendered on the screen. 

```suggestion
    Question: "Which of the following runtimes can be used for recursive functions that make multiple calls?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by deedeecee — COMMENTED** [review]
@jdwilkin4 Thankyou for your suggestion. I've made the requested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**